src/components/StockTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import { stockInfo } from '../../shared/stockInfo';
+import StockSymbol from '../StockSymbol';
+
+import './styles.css';
+import {
+    Table,
+    Thead,
+    Tbody,
+    Tr,
+    Th,
+    Td,
+    useColorModeValue,
+    VStack,
+    Container,
+    Text,
+    Code,
+} from '@chakra-ui/react';
+import React from 'react';
+import { useStockSymbolList } from '../../hooks/useStockList';
+
+const StockTable = () => {
+    const stocks = stockInfo;
+    const { stockDetails } = useStockSymbolList();
+    console.log('Stocks Context: ', stockDetails);
+    const colorScheme = useColorModeValue('brand.200', 'brand.800');
```

Comment:
This variable is just used in 1 place 
Can be used inline there

Reply:
Ok.

Reply:
On second thought, it's better to have it as a variable at the top rather than making it inline. Doesn't violate code smells that way.